src/org/rascalmpl/compiler/lang/rascalcore/check/ATypeUtils.rsc

```diff
@@ -1061,7 +1060,7 @@ Determine if the given type is an Abstract Data Type (ADT).
 bool isADTAType(aparameter(_,AType tvb)) = isADTAType(tvb);
 bool isADTAType(aadt(_,_,_)) = true;
 bool isADTAType(areified(_)) = true;
-bool isADTAType(\start(AType s)) = isADTAType(s);
```

Comment:
I have my doubts here. What does it mean to be a ADTAType? Should it be obly aadt's? Then this should be false for `start`. If it should be all left hand side of syntax rules, then it should be `true`.

Reply:
Perhaps @PaulKlint can answer this?